Modules/getpath.py

```diff
@@ -231,7 +231,8 @@ def search_up(prefix, *landmarks, test=isfile):
 pythonpath_was_set = config.get('module_search_paths_set')
 
 real_executable_dir = None
-stdlib_dir = None
+stdlib_dir = config.get('stdlib_dir')
```

Comment:
Probably move this up to one block, to all the other variables set by `config.get()`.